README.md

```diff
@@ -30,26 +30,14 @@ source .venv/bin/activate
 
 Install python dependencies:
 ```bash
-# Hivemind
-cd hivemind_source
-pip install .
-cp build/lib/hivemind/proto/* hivemind/proto/.
-pip install -e ".[all]"
-cd ..
-# Requirements
-pip install -r requirements.txt
-# Others
-pip install --pre torchdata --index-url https://download.pytorch.org/whl/nightly/cpu
-pip install -e ./pydantic_config
-# OpenDiLoCo
 pip install .
 ```
 
 Optionally, you can install flash-attn to use Flash Attention 2.
 This requires your system to have cuda compiler set up.
 ```
 # (Optional) flash-attn
-pip install flash-attn==2.5.8
```

Comment:
I dont think versions about 2.5.8 work. Theres some weird compiler version / shared library issue. could be wrong tho